rules/toolkit-base-windows.ham

```diff
@@ -0,0 +1,30 @@
+#----------------------------------------------------------------------
```

Comment:
See other comments, I'd start from `pierre-linux-clang` (PR #10) it should be very similar to this.